0053-bundle-format/README.md

```diff
@@ -0,0 +1,595 @@
+
+### User Stories
+
+#### Story 1: Offline Sign - Offline Verify
+
+As an operator deploying Zarf in an airgapped network, I need to verify package signatures without any external network connectivity while maintaining full cryptographic verification of package authenticity. This maintains backwards compatibility with existing keypair signing.
+
+**Solution**: Use airgap profile (default) with keypair signing.
+
+```bash
+# Developer (online environment) - sign package
+zarf package sign zarf-package-app-amd64.tar.zst --signing-key cosign.key
+
+# Operator (airgap environment) - verify
+zarf package verify zarf-package-app-amd64.tar.zst --key cosign.pub
+```
+
+#### Story 2: Online Sign - Offline Verify
+
+As a package author using Zarf in a connected environment, I want to leverage the Sigstore public good infrastructure to provide additional assurance that package signatures are publicly recorded and auditable.
+
+**Solution**: Use online profile with embedded Trusted Root verification.
+
+TODO: validate required inputs
+```bash
+# Developer - sign with transparency log upload
+zarf package sign zarf-package-app-amd64.tar.zst --signing-key cosign.key --profile online
+
+# Operator - verify with embedded trusted root - Do we need the public key?
+zarf package verify zarf-package-app-amd64.tar.zst
+```
+
+```bash
+# Developer - sign with keyless
+zarf package sign zarf-package-app-amd64.tar.zst --oidc-issuer='https://oauth2.sigstore.dev/auth' --profile online
+
+# Operator - verify with embedded trusted root - no external connectivity made
+zarf package verify zarf-package-app-amd64.tar.zst
+```
+
+#### Story 3: Online Sign - Online Verify - Private Sigstore Infrastructure
+
+As an enterprise running a private Sigstore instance for internal packages, I need to use a custom trusted root and private Fulcio/Rekor instances for signing and verification.
+
+**Solution**: Use online profile for signing and online verify with internal URLs or custom trusted root
+
+```bash
+
+# Sign with private infrastructure
+zarf package sign zarf-package-app-amd64.tar.zst \
+  --oidc-issuer https://oauth2.internal.company.com/auth \
+  --rekor-url https://rekor.internal.company.com \
+  --profile online
+
+# Verify with private infrastructure in an "online" profile
+Zarf package verify zarf-package-app-amd64.tar.zst \
+  --rekor-url https://rekor.internal.company.com \
+  --fulcio-url https://fulcio.internal.company.com \
+  --profile online
+
+# or verify in the default offline stance with a custom trusted-root
+# Create custom trusted root
+cosign trusted-root create \
+  --rekor-url https://rekor.internal.company.com \
+  --fulcio-url https://fulcio.internal.company.com \
+  --output custom_trusted_root.json
+
+# Verify with custom trusted root
+zarf package verify zarf-package-app-amd64.tar.zst \
+  --trusted-root custom_trusted_root.json
+```
+
+### Connectivity Profiles
+
+#### Airgap Profile (Default)
```

Comment:
The bullet point format and intermixed code make things a bit hard to follow IMO. Thoughts on a more paragraph style detailing each of the different options and how they apply to different use cases. And leaving the code blocks for the design details section